Services/Interfaces/web_interface/models/backtesting.py

```diff
@@ -109,23 +124,48 @@ def get_delete_data_file(file_name):
         return deleted, f"Can't delete {file_name} ({error})"
 
 
-def collect_data_file(exchange, symbol):
+def collect_data_file(exchange, symbol, start_timestamp=None, end_timestamp=None, in_background=False):
+    if in_background:
+        _background_collect_exchange_historical_data(exchange, symbol, start_timestamp, end_timestamp)
+        return True, f"Historical data collection started."
     success = False
     try:
```

Comment:
Could we also call `_collect_and_notify` here instead of duplicating this code?

Reply:
Not really because there is no notification here, it's a sync process and the `backtesting_api.collect_exchange_historical_data` is not called the same way in `_collect_and_notify` and here